src/client/containers/EventsFakeAPI.js

```diff
@@ -0,0 +1,103 @@
+// All properties are arrays of events
+// const EventsResponse = {
+//     "TODAY": [],
+//     "TOMORROW": [],
+//     "WEEK": [],
+//     "MONTH": []
+// }
+// Fake Api
+
+// Events Info for PageInfoComponent. Notice that it misses the color
+export const EventsPageInfo = {
+  // color: '#253F51',
+  title: 'EVENTS',
+  info: `Many events are organized throughout the semester. 
+    From companies' info sessions to simple workshops, 
+    we offer you many opportunities to become successful in your studies`,
+};
+
+// Might be moved to API where the weekdays are defined
+export const ProvidedTimes = [
+  'TODAY',
+  'TOMORROW',
+  'THIS WEEK',
+  'THIS MONTH',
+]
+
+export const EventField = [
+  'month',
```

Comment:
The way we declare constants in JavaScript is ...

```suggestion
const FIELD = {
    MONTH: 'month',
    FOO: 'foo',
    BAR: 'bar',
}
```

Reply:
Ok I changed it in the code